packages/apps/test/initSettings.spec.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright 2017-2021 @polkadot/apps authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { validateURL } from '../src/initSettings';
+
+describe('Test Init Function', () => {
+  test('should return URL for ws://localhost:9944', (done) => {
```

Comment:
Group all tests which return `true` and use `jest.each` to run them with 1 test.